docs/learn/benefits-lukso-standards.md

```diff
@@ -69,22 +116,81 @@ Some real-life examples for a user's Universal Profile could be:
+| **📢 Notify sender & recipient(s) on transfer and new operators**     | Sender is notified _"I have sent tokens"_, recipient is notified _"I have received tokens"_ and both can have custom logic to react on these notifications. <br/><br/>**Finished the old _"`approve(...)` then `transferFrom(...)`"_ flow!**. Using the Universal Receiver and automatic reactions on notifications is the new way! |
+| **⚙️ Extendability with pluggable Extensions**                        | New features (like new function selectors not present by default in the smart contract code) can be added to a Digital Asset, using a system of extensions.                                                                                                                                                                         |
+| **✋🏻 Safety parameter (by default!) to prevent accidental transfers** | Prevent bad web3 user stories and reduce the number of Google searches for:<br/> _"Transferred tokens to the wrong wallet address. How can I recover them?"_.                                                                                                                                                                       |
+
+### 🔘 Easier functions for developers
+
+Both [LSP7](../standards/tokens/LSP7-Digital-Asset.md) and [LSP8](../standards/tokens/LSP8-Identifiable-Digital-Asset.md) use the same function to transfer tokens, with only one different parameter type.
+
+- LSP7 [`transfer(address,address,uint256,bool,bytes)`](../contracts/contracts/LSP7DigitalAsset/LSP7DigitalAsset.md#transfer) function uses a **`uint256`** to transfer an **amount**.
+- LSP8 [`transfer(address,address,bytes32,bool,bytes)`](../contracts/contracts/LSP8IdentifiableDigitalAsset/LSP8IdentifiableDigitalAsset.md#transfer) function uses a **`bytes32`** to transfer a [**token ID**](../standards/tokens/LSP8-Identifiable-Digital-Asset.md#format-of-tokenids).
+
+Similarly, they both use the same function `authorizeOperator(address,[uint256/bytes32],bytes)` to approve new operators for a token allowance or specific token Ids.
+
+### 🗄️ Unlimited & Dynamic Metadata
+
+<div style={{textAlign: 'center', color: 'grey'}}>
+  <img
+    src={require('/img/learn/token-nft-metadata-examples.png').default}
+    alt="`Examples of metadata for different NFTs."
+    width="1200"
+  />
+<br/>
+<i>3 x different NFT collections with their unique traits and characteristics.</i>
+<br /><br />
+</div>
+
+Like Universal Profile, digital assets based on the LSP7 and LSP8 standards can hold an **unlimited quantity of metadata** for any various type of information considered relevant to the token or NFT collection. This can include:
```

Comment:
`any various type` - it feels like it should be either `any type` or `various type`. Reading `any various` sounds tautological, at least to me 🙂 

```suggestion
Like Universal Profile, digital assets based on the LSP7 and LSP8 standards can hold an **unlimited quantity of metadata** for any various type of information considered relevant to the token or NFT collection. This can include:
```